config.go

```diff
@@ -73,6 +76,10 @@ type config struct {
 	// AttributesGetter will be called to produce additional attributes while creating spans.
 	// Default returns nil
 	AttributesGetter AttributesGetter
+
+	// InstrumentAttributesGetter will be called to produce additional attributes while recording metrics.
```

Comment:
```suggestion
	// InstrumentAttributesGetter will be called to produce additional attributes while recording metrics to instruments.
```

Same as mentioned above.